### ISSUE TITLE
Doubling up on generating testcases with TestCase Importer

### ISSUE THREAD

**Issue by rarebreed** [RedHatQE/polarize#43]
I found several problems with the TestCase importer.  The first is while doing something like this:

```java
@TestDefinition( projectID = {Project.RHEL6, Project.RedHatEnterpriseLinux7} 
                            , testCaseID = {"", ""})
public void foo() {

}
```

It would generate 2 RedHatEnterpriseLinux7 TestCases instead of one.

**Comment by rarebreed** [comment]
This was being caused due to reuse of a Testcases object.  In the TestDefinitionProcessor object, it has a field called this.testcases.  Inside the Testcases object was a list of Testcase.  Since this object was being reused from this.testcase, and being used in initTestcases, each time initTestcases got called, it would use the now mutated value.

Solution is to not use this.testcases at all, and just pass in a fresh Testcases object to initTestcases.

**Comment by rarebreed** [comment]
Fixed in e08ee96
